KK_ButtPlugin/ButtPlugin.cs

```diff
@@ -42,6 +36,11 @@ private void Start()
                 defaultValue: 0,
                 "The difference in latency between your stroker and your display. \n" +
                 "Negative if your stroker has lower latency.");
+            EnableVibrate = Config.Bind(
+                section: "Device",
+                key: "Enable Vibrators",
+                defaultValue: true,

```

Comment:
This will generate 50 error logs per second if the device doesn't have vibration. So for the time being I would like you to turn this off by default.

Longer term we will probably need to get a DeviceList to auto detect capabilities.
```suggestion
                defaultValue: false,
```

Reply:
sgtm 👍   I was hoping to get a Device List added too so then I could vibrate all controllers instead of just device idx 0, but that should be outside the scope of this PR.